test/conftest.py

```diff
@@ -0,0 +1,89 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""System/E2E test setup"""
+import logging
+import subprocess
+from pathlib import Path
+
+import pytest
+
+pytest_plugins = "pytest_pixl"
+
+
+@pytest.fixture()
+def host_export_root_dir():
+    """Intermediate export dir as seen from the host"""
+    return Path(__file__).parents[1] / "exports"
+
+
+def run_subprocess(
```

Comment:
😎

Reply:
I think this function should live in an importable part of the package, so that we can re-use it in other places where we run subprocess. Hate to suggest `test/utils.py` but at least would be able to import it

Reply:
Can do - could it go in `pytest_pixl`, or is that not really for that?

Reply:
I think it could go in `pytest_pixl`, as long as it's not in `plugin.py` (as that is the entrypoint for the `pytest` plugin). Maybe something like `pytest_pixl/helpers.py`?

Reply:
Yep that sounds good to me.

Reply:
have moved it now